logger/meson.build

```diff
@@ -0,0 +1,3 @@
+logger_a = static_library('logger', 'src/main.cpp', include_directories
```

Comment:
I think we can do this in same existing meson.build.
until the team has some intention to keep it separate .